Grammar fixes (#40)
terra-money/terrain: README.md

---

> Terrain will help with the following:
> 
> - scaffolding your app: the contract and the frontend template
> - dramatically improve the development and deployment process
> 
> Terrain is not:
> 
> - a fully-featured Terra CLI. Take a look at [terrad](https://docs.terra.money/docs/develop/how-to/terrad/using-terrad.html).
> - an LCD itself. You will still need an rpc endpoint to deploy your contract. [LocalTerra](https://github.com/terra-money/LocalTerra) is one convenient option.

Reviewer: ```suggestion
Terrain help you:

- scaffold both the smart contract and the frontend of your app
- dramatically simplify the development and deployment process

Terrain is not:

- a fully-featured Terra CLI. Take a look at [terrad](https://docs.terra.money/docs/develop/how-to/terrad/using-terrad.html).
- an LCD. You will still need an RPC endpoint to deploy your contract. [LocalTerra](https://github.com/terra-money/LocalTerra) is a good development option for this.
```
